test/libsolidity/util/TestFunctionCallTests.cpp

```diff
@@ -103,6 +103,45 @@ BOOST_AUTO_TEST_CASE(format_hex_right_align)
 	BOOST_REQUIRE_THROW(test.format(), runtime_error);
 }
 
+BOOST_AUTO_TEST_CASE(format_bool_true_singleline)
+{
+	bytes expectedBytes = toBigEndian(u256{true});
+	ABIType abiType{ABIType::Boolean, ABIType::AlignRight, 32};
```

Comment:
I guess I'm a little confused about the size of the type in general. Shouldn't `bool` use only 1 byte in the ABI?

Reply:
The ABI specs says, that `bool` is encoded similarly to `uint8`. And for `uint8` we apply this (https://solidity.readthedocs.io/en/v0.5.4/abi-spec.html#formal-specification-of-the-encoding): 

> `uint<M>`: `enc(X)` is the big-endian encoding of X, padded on the higher-order (left) side with zero-bytes such that the length is 32 bytes.

And that makes it 32 bytes. Dooes that make sense to you?

Reply:
Ok, thanks!